### DISCUSSION TITLE
Please Release Recent version changes 1.1.1

### DISCUSSION THREAD

**Discussion by Shesharaopuli** [capacitor-community/google-maps#106] [General]
First of thanks a lot for the plugin and we really appreciate the efforts.

Right now the plugin is crashing because of map error. I have seen 
https://github.com/capacitor-community/capacitor-googlemaps-native/commit/4b93f55a41122ab054a7353bc720f25954764856

is solving the error. Could you please release 1.1.1 ?

**Comment by hemangsk** [comment]
Released 1.1.1 on npm

**Reply by Shesharaopuli** [reply]
I am able to get 1.1.1. But the code change is not applying.
![image](https://user-images.githubusercontent.com/14539389/134325745-5beab1fb-feda-4334-b086-f6b4fbcb86e0.png)

I tried uninstalling and installing , cleared cache. removed android and rebuild
but no luck.

Any suggestion would be appreciated

**Reply by hemangsk** [reply]
Please install 1.1.3 which we just released a few minutes ago. That resolves the problem of missing commits and should fix the issues which you're facing.

**Reply by Shesharaopuli** [reply]
https://github.com/capacitor-community/capacitor-googlemaps-native/discussions/108

Sure, So this can also be solved by updating to this version ? I mean that Fixed map element issue

**Reply by Shesharaopuli** [reply]
I have updated to 1.1.3 
/node_modules/@capacitor-community/capacitor-googlemaps-native/android/src/main/java/com/hemangkumar/capacitorgooglemaps/CapacitorGoogleMaps.java:457:
error: cannot find symbol
if (viewToRemove != null){
^
symbol: variable viewToRemove
Note:

i solved on my local and tried,  that fixed element problem is still giving 
So switching to https://github.com/DutchConcepts/capacitor-google-maps
but I dont understand how to use  reverseGeocodeCoordinate() in that plugin ?
